### PR TITLE
m-UpdateRegexForUpgradeConfig

### PR DESCRIPTION
**Pull request by blzaugg**

### CODE REVIEW

**Comment by robframpton** [comment]
Hey @blzaugg looks like the tests are failing, can you take a look?

**Comment by robframpton** [comment]
Looks the the tests are failing for #3 as well.

**Comment by robframpton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: test/tasks/upgrade/config.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ test.cb('upgrade:config', function(t) {
 		assert.fileContentMatch(pluginPackagePropertiesPath, /7\.0\.0\+/);
```
Comment (robframpton) [inline test/tasks/upgrade/config.js:58]: Hey @blzaugg 

This assertion is failing, it's looking for `7_0_0` in the look and feel file.

```
<!DOCTYPE look-and-feel PUBLIC "-//Liferay//DTD Look and Feel 7.0.0//EN" "http://www.liferay.com/dtd/liferay-look-and-feel_7.0.0.dtd">
```

That's what the file's doctype is after the upgrade task is run.

**Review by robframpton — CHANGES_REQUESTED** [review]

File: lib/upgrade/6.2/upgrade.js [hunk 2]
```diff
@@ -104,16 +104,20 @@ module.exports = function(options) {
...
-						replacement: '7.0.0+'
+						match: /(DTD Look and Feel )\d(?:\.\d+)+(\/\/EN)/g,
+						replacement: '$17.0.0$2'
 					},
 					{
-						match: /6\.2\.0/g,
-						replacement: '7.0.0'
+						match: /(liferay-look-and-feel_)\d(?:_\d+)+(\.dtd)/g,
+						replacement: '$17.0.0$2'
 					},
```
Comment (blzaugg) [inline lib/upgrade/6.2/upgrade.js:113]: I see the issue. I replaced with `'$17.0.0$2'`. I'll fix it.

Comment (blzaugg) [reply lib/upgrade/6.2/upgrade.js:113]: TODO: Replace with `'$17_0_0$2'`

**Comment by blzaugg** [comment]
https://github.com/Robert-Frampton/liferay-theme-tasks/pull/5

**Pull request closed without merging**